centos.sh

```diff
@@ -88,8 +90,8 @@ setup_network_config() {
 generate_config_mdadm() {
   if [ "$1" ]; then
     MDADMCONF="/etc/mdadm.conf"
-    echo "DEVICES /dev/[hs]d*" > $FOLD/hdd$MDADMCONF
-    echo "MAILADDR root" >> $FOLD/hdd$MDADMCONF
+    echo "DEVICES /dev/[hs]d*" > "$FOLD/hdd$MDADMCONF"
+    echo "MAILADDR root" >> "$FOLD/hdd$MDADMCONF"
     execute_chroot_command "mdadm --examine --scan >> $MDADMCONF"; EXITCODE=$?
     return $EXITCODE
```

Comment:
implemented:
```bash
declare -i EXITCODE="$?"
```

as mentioned in https://github.com/virtapi/installimage/pull/17#discussion-diff-50298170